daemon/commit.go

```diff
@@ -20,7 +21,10 @@ type ContainerCommitConfig struct {
 // The image can optionally be tagged into a repository.
 func (daemon *Daemon) Commit(container *Container, c *ContainerCommitConfig) (*image.Image, error) {
 	if c.Pause && !container.isPaused() {
-		container.pause()
+		if err := container.pause(); err != nil && !strings.Contains(err.Error(), "is not running") && strings.Contains(err.Error(), "is already paused") {
```

Comment:
How about use error type in `github.com/docker/docker/errors`, like ErrorCodeNotRunning and ErrorCodeAlreadyPaused